src/_utils/focusVisibleProvider/index.unit.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react'
+import { act } from 'react-dom/test-utils'
+import { mount } from 'enzyme'
+
+import FocusVisibleProvider, { FocusVisibleContext } from '.'
+import { KEY } from '_utils/keycodes'
+
+const ChildComponent = () => {
+  const hadKeyboardEvent = useContext(FocusVisibleContext)
+  return <input type="button" className={hadKeyboardEvent.toString()} value="Test" />
+}
+
+describe('FocusVisibleProvider', () => {
+  let parentNode
+  let wrapper
+  beforeEach(() => {
+    parentNode = document.createElement('div')
+    document.body.appendChild(parentNode)
```

Comment:
Isn't this already done by enzyme?

Reply:
I do it this way not to refer directly to `document.body` when I attach it to the mount() function. I followed this example:
https://fr.reactjs.org/docs/test-utils.html
And I need to refer to a global element to fire events.

Reply:
I see, I think they use this in because they rely on `ReactDOM.render()` in their tests.
Here we currently use enzyme to handle the rendering and looking at their doc we don't need it.

For the event, doesn't `document.body.dispatchEvent()` work?

Reply:
Oh you right, no need to attach anything to the mount() function.
I previously try:
```
wrapper = mount(
      <FocusVisibleProvider>
        <ChildComponent />
      </FocusVisibleProvider>,
      { attachTo: document.body },
    )
```
Witch works but trigger some warnings.
Anyway your solution is cleaner :-)